jekyll/_cci2/server/installation/phase-4-post-installation.adoc

```diff
@@ -523,6 +523,915 @@ Server installations include their own local orb registry. This registry is priv
 
 For more information and steps to complete these tasks, see the link:/docs/server/operator/managing-orbs[Orbs on server guide].
 
+[#all-values-yaml-options]
+== All Helm values.yaml Options
+
+[options="header"]
+|===
+|Key |Type |Default |Description
+|apiToken
+|string
+|`""`
+|API token (2 Options). Option 1: Set the value here and CircleCI will create the secret automatically. Option 2: Leave this blank, and create the secret yourself. CircleCI will assume it exists.
+|api_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the api-service deployment.
+|audit_log_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the audit-log-service deployment.
+|branch_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the branch-service deployment.
+|builds_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the builds-service deployment.
+|contexts_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the contexts-service deployment.
+|cron_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the cron-service deployment.
+|dispatcher.replicas
+|int
+|`1`
+|Number of replicas to deploy for the dispatcher deployment.
+|distributor_cleaner.replicas
+|int
+|`1`
+|Number of replicas to deploy for the distributor-dispatcher deployment.
+|distributor_dispatcher.replicas
+|int
+|`1`
+|Number of replicas to deploy for the distributor-dispatcher deployment.
+|distributor_external.replicas
+|int
+|`1`
+|Number of replicas to deploy for the distributor-external deployment.
+|distributor_internal.replicas
+|int
+|`1`
+|Number of replicas to deploy for the distributor-internal deployment.
+|domain_service.replicas
+|int
+|`1`
+|Number of replicas to deploy for the domain-service deployment.
+|frontend.replicas
+|int
+|`1`
+|Number of replicas to deploy for the frontend deployment.
+|github
+|object
+| `{
+  "clientId": "",
+  "clientSecret": "",
+  "enterprise": false,
+  "fingerprint": null,
+  "hostname": "ghe.example.com",
+  "scheme": "https",
+  "selfSignedCert": false,
+  "unsafeDisableWebhookSSLVerification": false
+}`
+|VCS Configuration details (currently limited to Github Enterprise and Github.com)
+|github.clientId
+|string
+|`""`
+|Client ID for OAuth Login via Github (2 Options). Option 1: Set the value here and CircleCI will create the secret automatically. Option 2: Leave this blank, and create the secret yourself. CircleCI will assume it exists. Create on by Navigating to Settings > Developer Settings > OAuth Apps. Your homepage should be set to `{{ .Values.global.scheme }}://{{ .Values.global.domainName }}` and callback should be `{{ .Value.scheme }}://{{ .Values.global.domainName }}/auth/github`.
```

Comment:
This table is currently breaking the docs build, I think because of the `{{ }}` sections here. Is this exactly as it needs to be shown?

Reply:
Maybe we can do `$()` instead of `{{ }}`?

Reply:
Ah thanks. I've just worked out a fix but the table still looks gross so needs a bit more work!